parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -424,9 +428,11 @@ def __init__(
         context_generator: Optional[ContextGenerator] = None
         if (
             args.blueprint.include_persona
-            or args.blueprint.conversation_start_mode == 'bst'
+            or args.blueprint.conversation_start_mode != 'hi'
```

Comment:
Add: `# 'hi' mode does not use a context generator and instead just displays "Hi!" at the start of the conversation`